src/style.css

```diff
@@ -1,3 +1,72 @@
+html{
+  background-color: wheat;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  size: 12px;
```

Comment:
The 'size' property is not a valid CSS property. If you intend to set the font size, use 'font-size' instead.